cmd/clusterctl/pkg/client/repository/templates_client.go

```diff
@@ -61,7 +52,31 @@ func newTemplatesClient(provider config.Provider, version string, repository Rep
 
 // Get return the template for the flavor/bootstrap provider specified.
 // In case the template does not exists, an error is returned.
-func (f *templatesClient) Get(flavor, bootstrap string, options TemplateOptions) (Template, error) {
-	//TODO: implement in a follow up PR
-	return nil, nil
+// Get assumes a naming following naming convention fo templates: config[-<flavor_name>]-<bootstrap_provider_name>.yaml
+func (f *templatesClient) Get(flavor, bootstrap, targetNamespace string) (Template, error) {
```

Comment:
nit: (t *templatesClient)  or (c *templatesClient). f seems to suggest flavor.

Reply:
done